source/docs/zero-to-robot/step-3/radio-programming.rst

```diff
@@ -2,198 +2,133 @@
 
 # Programming your Radio
 
-This guide will show you how to use the FRC\ |reg| Radio Configuration Utility software to configure your robot's wireless bridge for use outside of FRC events.
+This guide will show you how to do a basic setup for controlling your robot "at home" using the VH-109 FRC\ |reg| Radio.  For complete documentation and information please see the [Vivid-Hosting radio site](https://frc-radio.vivid-hosting.net/).
 
 ## Prerequisites
 
-The FRC Radio Configuration Utility requires administrator privileges to configure the network settings on your machine. The program should request the necessary privileges automatically (may require a password if run from a non-administrator account), but if you are having trouble, try running it from an administrator account.
+The recommended setup requires: 2 VH-109 radios, 1 VH-117 :term:`PoE` Wall Adapter, and 1 Radio Heatsink.  Available [here](https://wcproducts.com/products/frc-radio).
```

Comment:
This feels like it could really use a picture to show how everything works together.

Reply:
Are you talking about a picture like those here?
https://frc-radio.vivid-hosting.net/getting-started/usage/wiring-your-radio

Or here:
https://frc-radio.vivid-hosting.net/getting-started/usage/practicing-at-home#connectivity-diagram

I don't like the latter because it incorrectly labels the AP radio the robot radio and it shows an unnecessary unmanaged switch that the vast majority of teams don't need.

Reply:
The latter is what I was thinking of